src/backend/orchestration/index.ts

```diff
@@ -75,6 +76,11 @@ export interface OrchestrationProps {
    */
   readonly monitoring: Monitoring;
 
+  /**
+   * The onCall dashboard to register dashboards with.
+   */
+  readonly onCallDashboard: OverviewDashboard;
```

Comment:
It looks like we're referring to this as both the oncall dashboard and overview dashboard -- we should probably just use one term everywhere and be consistent since this is technically part of the public API. This appears in a few couple other places as well
```suggestion
  readonly overviewDashboard: OverviewDashboard;
```